app/script.js

```diff
@@ -0,0 +1,13 @@
+$(window).resize(function(){
```

Comment:
This script can be replaced with the usage of media-queries.

You could use, for instance:

```@media only screen and (max-width : 767px) {
  #login, #register, #figures {
    display: none;
  }
}```